Yoshi/Yoshi/YoshiInvocation.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  YoshiInvocation.swift
+//  Pods
+//
+//  Created by Quentin Ribierre on 10/24/16.
+//
+//
+
+import Foundation
+
+/// Set of Yoshi invocation method.
+public struct YoshiInvocation: OptionSetType {
+
+    public let rawValue: UInt64
+
+    public init(rawValue: UInt64) {
+        self.rawValue = rawValue
+    }
+
+    /// All invocations method.
+    public static let all = YoshiInvocation(rawValue: 0)
+
+    /// Shake motion gesture invocation method.
+    public static let shakeMotionGesture = YoshiInvocation(rawValue: 1 << 0)
+    /// Multi touch event invocation method.
+    public static let multiTouch = YoshiInvocation(rawValue: 1 << 1)
+    /// Force touch event invocation method.
+    public static let forceTouch = YoshiInvocation(rawValue: 1 << 2)
```

Comment:
we should have consistent spacing between the invocation types

- [ ] fix spacing